data/mods/Xedra_Evolved/items/armor/integrated.json

```diff
@@ -793,5 +793,21 @@
       }
     ],
     "armor": [ { "covers": [ "torso" ], "coverage": 0, "encumbrance": 0 } ]
+  },
+  {
+    "id": "integrated_nobreathe",
+    "type": "ARMOR",
+    "category": "armor",
+    "name": { "str_sp": "vestigial lungs" },
+    "description": "Breathing is now something you decide to do, rather than a biological necessity.",
```

Comment:
Does this autoset oxygen levels so that you can't asphyxiate from being crushed to death?

Reply:
I first thought that the oxygen value were hardcoded

Then I checked and the REBREATHER flag exists and prevents being suffocated by crushing so I'll add it to the lungs

The messages are hardcoded however, so they'll get told that they are suffocating regardless

Reply:
Ok that’ll create an issue that will hopefully result in oxygen getting less hardcoded.